roles/basic-site/tasks/main.yml

```diff
@@ -31,6 +31,8 @@
   copy:
     dest: /var/www/{{ site }}/cert/dns
     content: |
+      # This file is managed by ansible. It belongs to {{ role_path }}.
```

Comment:
Does `/opt/letsencrypt/update_keys.sh` handle commented lines? I'm quite sure that it does not.

Reply:
it does now :)